python/ray/tune/analysis/post_experiment.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import logging
+import os
+import os.path as osp
+import numpy as np
+import pandas as pd
+from pandas.api.types import is_string_dtype, is_numeric_dtype
+from ray.tune.util import flatten_dict
+
+logger = logging.getLogger(__name__)
+
+
+def get_best_trial(trial_list, metric):
+    """Retrieve the best trial."""
+    return max(trial_list, key=lambda trial: trial.last_result.get(metric, 0))
+
+def get_sorted_trials(trial_list, metric):
+    return sorted(trial_list, key=lambda trial: trial.last_result.get(metric, 0), reverse=True)
+
+def get_best_result(trial_list, metric):
+    """Retrieve the last result from the best trial."""
+    return {metric: get_best_trial(trial_list, metric).last_result[metric]}
+
+
+def _parse_results(res_path):
+    res_dict = {}
+    try:
+        with open(res_path) as f:
+            # Get last line in file
+            for line in f:
+                pass
+        res_dict = flatten_dict(json.loads(line.strip()))
+    except Exception:
+        logger.exception("Importing %s failed...Perhaps empty?" % res_path)
+    return res_dict
+
+def _parse_configs(cfg_path):
+    try:
+        with open(cfg_path) as f:
+            cfg_dict = flatten_dict(json.load(f))
+    except Exception:
+        logger.exception("Config parsing failed.")
+    return cfg_dict
+
+def _resolve(directory, result_fname):
+    try:
+        resultp = osp.join(directory, result_fname)
```

Comment:
I would use full names for variables like these:
```suggestion
        result_path = osp.join(directory, result_filename)
```